docs/docs/examples/guides/ur-harware-interface.md

```diff
@@ -0,0 +1,577 @@
+control rate of the real hardware.
+
+:::
+
+<div class="text--center">
+  <img src={urHwiStudio} alt="UR Hardware Interface in AICA Studio" />
+</div>
+
+<!-- TO ADD, TBD:
+
+- use of the dashboard controller with local mode and the program node to hand back control between AICA and UR
+- use of the UR impedance controller and hand guiding controller to leverage UR Force mode from AICA Studio
+- use of the dashboard server to observe GPIOs, set payload, zero ft sensor etc -->
+
+<!-- # Hardware Interface
+
+TBD: There is already a [page](../../concepts/building-blocks/hardware-interfaces.md) on hardware interfaces. Are these
+two conflicting? Should we link to this one?
+
+Within the context of AICA Studio, but also in ROS, hardware interfaces are used as middleware between controllers and
+actual hardware. In other words, they are tasked with two-way communication of commands and state feedback between the
+robot and the controller that usually lies above. -->
+
+## Local and Remote Control
+
+The concept of Local and Remote Control on PolyScope can be easiest explained by introducing a _primary_ and _secondary_
+device architecture. In Local Control, the controller is the _primary_ and has full authority on loading and starting
+programs. In other words, the robot has to be used in person through the teach pendant and any commands sent from an
+external source will be rejected. On the other hand, Remote Control allows to control the robot via external sources,
+such as sockets, I/Os and the Dashboard Server. In this case, the controller is the _secondary_ and external sources can
+load and start programs or directly send URScript commands to the controller.
+
+:::note
+
+Safety features remain active in Remote Control.
+
+:::
+
+Choosing one of the two modes depends on the specific task at hand. During a development phase, it might be preferable
+to create the programs in Local Mode, whereas in a production setting, PLCs would be responsible to load and start the
+desired programs while the robot is in Remote Control. With the AICA System, users have the possibility to get the best
+of both modes:
+
+1. Take full control of the robot from an AICA application (requires Remote Control)
+2. Run an AICA application as one node of a program (works in both Local and Remote Control)
+
+### Full control of the robot from an AICA application
+
+For this first, simpler case, no additional installation steps are required. Apart from setting the correct robot IP,
+two requirements have to be met:
+
+- On the robot, Remote Control has to be activated. For that, first activate Remote Control in the system settings as
+  explained [here](./ur-sim-guide.md#accessing-and-configuring-the-simulated-robot), then switch from _Local_ to
+  _Remote_ mode on the top right corner of the teach pendant. The interface automatically switches to the _Run_ tab and
+  disables other tabs, indicating that control has been handed over to external sources.
```

Comment:
```suggestion
  disables other tabs, indicating that control is ready to be handed over to external sources.
```

nit: again, not the best replacement, but I feel like the handover is complete closer to when we have "Running" (i.e., it can me in remote but no one is connected to it)

Reply:
hmm, yes fair enough